sdks/wasm/README.md

```diff
@@ -101,28 +101,36 @@
 * The makefile targets are in sdks/builds/wasm.mk.
 * The build outputs are emitted to sdks/out/.
 
+## Requirements
+
+WebAssembly builds require Ninja in addition to the standard Mono dependencies. On MacOS, this can be installed with `brew install ninja`.
+
+```ulimit -n 2048```
```

Comment:
Any reasoning for `2048`?  Do you mind updating the actual check here? https://github.com/mono/mono/blob/04c01efd4997d1f58ecdc0310932df2f4eb84fe5/sdks/builds/Makefile#L81

Reply:
Oh, I think I meant to put 1024 >_>

The MacOS default is 256 or something, which is too low, but 1024 should be fine.